ui/jobs/jobs.js

```diff
@@ -267,6 +281,10 @@ class ComboTracker {
   AbortCombo() {
     this.StateTransition(null);
   }
+  IsComboBroken() {
+    if (this.considerNext == this.startList) return true;
```

Comment:
I think this can be `return this.considerNext === this.startList;`

But yeah, this is exactly what I meant, thanks!